Formula/omniorb.rb

```diff
@@ -19,6 +19,7 @@ class Omniorb < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "python"
```

Comment:
```suggestion
  depends_on "python@3.9"
```
The formula is called `python@3.9` now.

Reply:
Must it be tied to a specific version?  Its bindings work with any version of Python 3.

I remember how long it took to update to Python 3.8, I don't want this to get in the way of the next update unless it's necessary.

Reply:
`depends_on "python"` means the same thing, just implicitly. The slow migration to python 3.8 is actually the reason this needs to be specified like this. It allows us to move the `python` alias without breaking everything that depends on "any python"